astropy/units/quantity_helper/function_helpers.py

```diff
@@ -437,29 +443,28 @@ def _block(arrays, max_depth, result_ndim, depth=0):
     else:
         return np_core.shape_base._atleast_nd(arrays, result_ndim)
 
+_UnitFromLikeArg = object()
```

Comment:
Why the camel case? I thought that was meant more for classes. Just `unit_from_like` or `UNIT_FROM_LIKE` (no real reason for it to be private, I think; none of these are exported up).

Reply:
> Why the camel case?

I don't know 😅